docs/integrations/sources/mssql.md

```diff
@@ -204,21 +206,61 @@ For further detail, see the [Microsoft docs on enabling and disabling CDC](https
 
 #### Ensuring the SQL Server Agent is running
 
-- MSSQL uses the SQL Server Agent to [run the jobs necessary](https://docs.microsoft.com/en-us/sql/relational-databases/track-changes/about-change-data-capture-sql-server?view=sql-server-ver15#agent-jobs) for CDC. It is therefore vital that the Agent is operational in order for to CDC to work effectively. You can check the status of the SQL Server Agent as follows:
+- MSSQL uses the SQL Server Agent
+  to [run the jobs necessary](https://docs.microsoft.com/en-us/sql/relational-databases/track-changes/about-change-data-capture-sql-server?view=sql-server-ver15#agent-jobs)
+  for CDC. It is therefore vital that the Agent is operational in order for to CDC to work effectively. You can check
+  the status of the SQL Server Agent as follows:
+
 ```text
   EXEC xp_servicecontrol 'QueryState', N'SQLServerAGENT';
 ```
-- If you see something other than 'Running.' please follow the [Microsoft docs](https://docs.microsoft.com/en-us/sql/ssms/agent/start-stop-or-pause-the-sql-server-agent-service?view=sql-server-ver15) to start the service.
 
-#### Setting up CDC on managed versions of SQL Server
+- If you see something other than 'Running.' please follow
+  the [Microsoft docs](https://docs.microsoft.com/en-us/sql/ssms/agent/start-stop-or-pause-the-sql-server-agent-service?view=sql-server-ver15)
+  to start the service.
 
-We readily welcome [contributions to our docs](https://github.com/airbytehq/airbyte/tree/master/docs) providing setup instructions. Please consider contributing to expand our docs!
+#### Setting up CDC on managed versions of SQL Server
 
+We readily welcome [contributions to our docs](https://github.com/airbytehq/airbyte/tree/master/docs) providing setup
+instructions. Please consider contributing to expand our docs!
+
+## Connection to MSSQL via an SSH Tunnel
+
+Airbyte has the ability to connect to a MSSQL instance via an SSH Tunnel. The reason you might want to do this because
+it is not possible (or against security policy) to connect to the database directly (e.g. it does not have a public IP
+address).
+
+When using an SSH tunnel, you are configuring Airbyte to connect to an intermediate server (a.k.a. a bastion sever)
+that _does_ have direct access to the database. Airbyte connects to the bastion and then asks the bastion to connect
+directly to the server.
+
+Using this feature requires additional configuration, when creating the source. We will talk through what each piece of
+configuration means.
+
+1. Configure all fields for the source as you normally would, except `SSH Tunnel Method`.
+2. `SSH Tunnel Method` defaults to `No Tunnel` (meaning a direct connection). If you want to use an SSH Tunnel
+   choose `SSH Key Authentication` or `Password Authentication`.
+1. Choose `Key Authentication` if you will be using an RSA Private as your secrets for establishing the SSH Tunnel (see
```

Comment:
Will copy-paste the comment I'd got from Sherif for this part:
" Choose `Key Authentication` if you will be using an RSA Private as your secrets for establishing the SSH Tunnel (see below for more information on generating this key)."  ->  "Choose `Key Authentication` if you will be using an RSA private key as your secret for establishing the SSH Tunnel (see below for more information on generating this key)."

Reply:
thank you, done